org.eclipse.winery.lsp/src/test/resources/testArtifactType.yaml

```diff
@@ -0,0 +1,5 @@
+    description: >
```

Comment:
I think, this is not valid TOSCA. :)

Please read the intro of section 6 and section 6.1, section 6.2, and Section 6.4.1 of the TOSCA specification. :)

```yaml
tosca_definitions_version: tosca_2_0

artifact_types:
    python:
```

You can find some other TOSCA examples at https://github.com/tliron/puccini/tree/main/examples/tosca -- they should roughly also be convertable to TOSCA 2.0.

Especially https://github.com/tliron/puccini/blob/main/examples/tosca/artifacts.yaml is interesting in this case.

Reply:
It's not clear to me which keynames are considered mandatory in the scope of artifact type entity, from the TOSCA specification it appears that there are no mandatory fields.
- Also the Common Keynames in Type Definitions in section 6.4.1, are they considered optional or mandatory in the scope of artifact type entity?